projects/go-lib/src/lib/components/go-off-canvas/go-off-canvas.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from "@angular/core";
+import { Subject } from 'rxjs/internal/Subject';
+import { GoOffCanvasItem } from "./go-off-canvas.interface";
+
+@Injectable()
```

Comment:
Yes, you are correct. If I understand this correctly, we should almost always have the `@injectable` configured to provide itself in the `'root'`. This will help tell Angular how to manage the tree shaking it does when we compile things for production.